starknet_py/compile/compiler_test.py

```diff
@@ -0,0 +1,25 @@
+import json
+from starkware.starknet.services.api.contract_definition import ContractDefinition
+
+from starknet_py.compile.compiler import Compiler
+from starknet_py.compile.map import MAP_CONTRACT
```

Comment:
I think it'd be best to avoid adding compiled cairo code to the repo, and we can re-use map contract from `starknet_py/utils/compiler` module